### PR TITLE
Create acceptStudent function

### PR DESCRIPTION
**Pull request by amsolaiman85**
- acceptStudent function is created.
- Key value 201812485 = ['Abdul Moiz Solaiman', 22, 'Marawi City'] is stored.
- Status output is set to "Applying."

### CODE REVIEW

**Comment by amsolaiman85** [comment]
This marks Issue #3 as done.

File: main.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
 var level = require('level');
 
-var db = level('./leveldb');
\ No newline at end of file
+var db = level('./leveldb');
+
+acceptStudent(['Abdul Moiz Solaiman', 22, 'Marawi City']);
+
+function acceptStudent(attributes){
```
Comment (fencim) [inline main.js:7]: the function signature should accept similar to mentioned in Issue 
`function acceptStudent(id, fullname, age, address)`

File: main.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    db.put('201812485', attributes, function(err){
```
Comment (fencim) [inline main.js:8]: this can't be hard-coded, id should passed as  argument as specified in Issue

**Review by fencim — COMMENTED** [review]
create separate pull request for comments, since its already merged

**Pull request merged**